# Please contact us anytime for details or troubleshooting. Happy to help.



This is the list of files that are part of this repo. Each file contains detailed comments/troubleshoot tips/corresponding figure/data identifier along with associated code.

1. Data Density Scatter
2. Distribution Scaling
3. Multi_range_rows
4. Dim. reduction - 3D
5. Hierarchical clustering
6. Correlation matrices for multidim. data
7. Operations_large_file
8. Testing_intergroup_variations
9. t-SNE (t-distributed stochastic neighborhood embedding)
91. test_datafits
94. k-means_clustering
95. Biological_networks_cytoscape

More details can be provided upon request.

Technical Requirements:

Softwares, Languages et al:

Data Collection: Image Quant 5.2 (quantification of crosslinking timepoints), Image J 1.8.0 (quantification of western blots, in vivo bulk crosslinking)

Data Analysis: KinTek Explorer 10.0.200513 (in vitro data fitting), FASTX-Toolkit 0.0.13 (NGS read processing), FastQC 0.11.9 (NGS read quality),  Microsoft EXCEL 2019 16.0.6742.2048 (data arrangement and manipulation), Bowtie2 2.4.2 (NGS read mapping), IGV genome browser 2.8x (NGS data visualization), Cytoscape 3.4.0 (biological networks), Weblogo 2.8.2 (NGS iCLIP, sequence logo), Kaleidagraph 4.1.1 (plots), Bedtools 2.29.1 (NGS read peak calling), Samtools 1.10 (NGS read processing), UCSC genome browser 1635.2 (NGS data visualization), Emboss Compseq 6.0.0 (motif enrichment), Python 3.9.0 (for data fitting, PCA, tSNE, heatmaps, boxplots, ANOVA), Additional python modules: random 3.9.0  (motif enrichment and scrambling), scipy.hypergeom 3.6.5 (hypergeometric enrichment tests),  Scikit-learn 0.23.0 (multiple linear regression analysis), Statsmodel 0.11.0 (multiple linear regression model permutations), CHAID 5.3.0 (CHAID analysis), R Project For Statistical Computing 4.3.0 (data normalization, data fitting and data visualization), Additional R modules: prcomp 3.6.2 (Principal Component Analysis), vignette 3.6.2 (Monte Carlo Simulation), car 3.0.10 (one-way ANOVA), deSolve 1.28 (solving differential equations), ggplot2 3.3.2 (visualization - heatmaps, density plots), reshape21.4.4 (data normalization and transformation), rmarkdown 2.6.0 (data formatting), randomizeR 2.0.0 (generating random sequences), rtSNE 0.13.0 (T-Distributed Stochastic Neighbor Embedding using a Barnes-Hut Implementation)

Additional base requirements for specific steps: Perl (Strawberry) - 5.32.0
C++ with gcc 7.2 compiler, C (check your computer manual), Cygwin for Windows - 3.1, Bash - 5.0, Windows Command Prompt (check your computer manual), Tensorflow - Core v2.3.0
.
